packages/slic-tools/dynamo.js

```diff
@@ -0,0 +1,41 @@
+const { DynamoDBClient } = require('@aws-sdk/client-dynamodb')
```

Comment:
rename this file `aws.js` to `dynamo.js` since this file now only exports a dynamo client